### PR TITLE
l10n: Align source strings

### PR DESCRIPTION
**Pull request by rakekniven**
Signed-off-by: rakekniven <2069590+rakekniven@users.noreply.github.com>

### CODE REVIEW

**Review by Valdnet — APPROVED** [review]

**Comment by nickvergessen** [comment]
I think the source string needs to be adjusted not the translation, otherwise it is just reverted on the next run?
Please in apps that are not plain php/js always wait for the maintainer to confirm and merge changes

**Pull request merged**
